src/schoology-extractor/schoology_extractor/main.py

```diff
@@ -50,7 +56,9 @@
 
 for section_id in sections_id_array:
     submissions_response = request_client.get_submissions_by_section_id(section_id)
+
     while True:
+        print(submissions_response.current_page_items)
```

Comment:
For the logging story: good example of something that might be a DEBUG log entry.